### DISCUSSION TITLE
Classic/Premium 50.04 Support milestone.

### DISCUSSION THREAD

**Discussion by ZehCastro** [DFHack/dfhack#2550] [Releases]
There are no [milestones](https://github.com/DFHack/dfhack/milestones) tracking 0.50.04 support.
Shouldn't we have one, or some other way to track that?

**Comment by lethosor** [comment]
We use GitHub projects for this, and there is one named "next release" that tracks 50.x support: https://github.com/orgs/DFHack/projects/22
